Docs for palettes and aes (#108)
posit-dev/ggsql: doc/syntax/scale/aesthetic/shape.qmd

---
title: Shape
---

> `shape` governs how points look, wherever they may appear (usually in a point layer). Shape can be used instead of color to differentiate between categories of points (or even better, in concert with color). Shape is only meaningful in relation to discrete data.

Reviewer: ```suggestion
The `shape` aesthetic governs how points look. Shape can be used instead of color to differentiate between categories of points (or even better, in concert with color). Shape is only meaningful in relation to discrete data.
```
